distribution/docs/src/main/resources/content/_configuring/disable-security-manager.adoc

```diff
@@ -0,0 +1,25 @@
+:title: Disabling Security Manager
+:type: subConfiguration
+:status: published
+:parent: Setting Security Manager Policies
+:summary: Setting security manager policies.
+:order: 00
+
+== {title}
+
+To disable the security manager entirely, edit the `setenv` file located in `${branding}_HOME/bin`.
```

Comment:
[PR 2447](https://github.com/codice/ddf/pull/2447) will be adding 2 new setenv files: 
- setenv-wrapper.conf
- setenv-windows-wrapper.conf 

which requires
```wrapper.java.additional.nnn = -Djava.security.policy==%KARAF_HOME%/security/default.policy
wrapper.java.additional.nnn = -Dpolicy.provider=net.sourceforge.prograde.policy.ProGradePolicy
wrapper.java.additional.nnn = -DproGrade.getPermissions.override=sun.rmi.server.LoaderHandler:loadClass,org.apache.jasper.compiler.JspRuntimeContext:initSecurity
wrapper.java.additional.nnn = -Djava.security.manager=net.sourceforge.prograde.sm.ProGradeJSM``` (nnn are numbers that might change) to be commented out .